lib/util/device-service-details.ts

```diff
@@ -117,11 +119,7 @@ export const generateCurrentServiceDetails = (
 		getSingleInstallSummary(gd),
 	);
 
-	// prefer over omit for performance reasons
-	delete rawDevice.image_install;
-	delete rawDevice.gateway_download;
-
-	const device = (rawDevice as Device) as DeviceWithServiceDetails;
+	const device = rawDevice as DeviceWithServiceDetails;
```

Comment:
I'd move the assignment of this var to the bottom and have
```ts
	const device = rawDevice as DeviceWithServiceDetails;
	device.current_services = currentServicesGroupedByName;
	device.current_gateway_downloads = downloads;
	return device;
```
in order to reduce the amount of code it exists for and make it clearer that it's just a reassignment in order to add a couple of properties

Reply:
:+1: